diesel_derives/src/util.rs

```diff
@@ -1,14 +1,13 @@
-use proc_macro2::Span;
-use quote::Tokens;
+use proc_macro2::*;
 use syn::*;
 
 pub use diagnostic_shim::*;
 use meta::*;
 
-pub fn wrap_in_dummy_mod(const_name: Ident, item: Tokens) -> Tokens {
+pub fn wrap_in_dummy_mod(const_name: Ident, item: TokenStream) -> TokenStream {
     quote! {
         #[allow(non_snake_case, unused_extern_crates, unused_imports)]
-        mod #const_name {
```

Comment:
Hu ? This isn't a dummy mod then

Reply:
Sure -- Just didn't really feel like churning every call. Probably should have named it `dummy_scope` or something since we've now changed it from const to mod to fn